mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModel.kt

```diff
@@ -416,62 +412,19 @@ abstract class MavericksViewModel<S : MavericksState>(
         deliveryMode: DeliveryMode,
         action: suspend (T) -> Unit
     ): Job {
-        val flow = if (lifecycleOwner == null || MavericksTestOverrides.FORCE_DISABLE_LIFECYCLE_AWARE_OBSERVER) {
-            this
-        } else if (deliveryMode is UniqueOnly) {
-            val lastDeliveredValue: T? = lastDeliveredValue(deliveryMode)
-            this
-                .assertOneActiveSubscription(lifecycleOwner, deliveryMode)
-                .dropWhile { it == lastDeliveredValue }
-                .flowWhenStarted(lifecycleOwner)
-                .distinctUntilChanged()
-                .onEach { lastDeliveredStates[deliveryMode.subscriptionId] = it }
+        return if (lifecycleOwner != null) {
```

Comment:
would it be worth making `collectLatest(lifecycleOwner,` nullable so we can pass null here and not duplicate this launch logic?

Reply:
hmm im thinking with collectLatest we would want it the owner be explicitly sent in since it just an extension/not tied to something like a VM which could have fallback logic